contrib/gce/configure.sh

```diff
@@ -104,6 +104,26 @@ if [ -f "${CONTAINERD_HOME}/${CONTAINERD_ENV_METADATA}" ]; then
   source "${CONTAINERD_HOME}/${CONTAINERD_ENV_METADATA}"
 fi
 
+# cgroupv2 setup
+if [ "${CONTAINERD_CGROUPV2:-"false"}"  == "true" ]; then
+  if uname -a | grep -q cos; then
+    if ! grep -q 'systemd.unified_cgroup_hierarchy=true' /proc/cmdline; then
+      echo "Setting up cgroupv2"
+
+      mount_path="/tmp/esp"
```

Comment:
this is a "magic" path? can it be anything? `esp` ... and what is ESP?

Reply:
ESP mean EFI System Partition. It containers boot loader and some files for vm to use at boot time. 

More detail could refer: https://en.wikipedia.org/wiki/EFI_system_partition

Reply:
I don't think `/tmp/esp` is special, the mount path can be any directory. The only special "magic" thing here is `/dev/sda12`.

Reply:
thanks @bobbypage and @Dragoncell !